lib/client.js

```diff
@@ -209,12 +239,11 @@ module.exports = class EtekCityClient {
             uri = '/powerstrip/v1/device/devicestatus';
         }
         // Smart Light Bulbs
-        else if (device.type.startsWith('ESL100') || 
-                 device.type === 'ESL100CW' || device.type === 'ESL100MC') {
+        else if (SUPPORTED_LIGHT_BULBS.includes(device.type)) {
             uri = '/smartlight/v1/device/devicestatus';
         }
 
-        return this.client.put(uri, {
+        return this.client.post(uri, {
```

Comment:
assuming some scenarios will need to be a `put` but this `post` matches what pyvesync was doing